addons/l10n_latam_check/models/l10n_latam_check.py

```diff
@@ -0,0 +1,217 @@
+# pylint: disable=protected-access
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+import stdnum
+
+from odoo import models, fields, api, Command, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools import index_exists
+
+
+_logger = logging.getLogger(__name__)
+
+
+class l10nLatamAccountPaymentCheck(models.Model):
+    _name = 'l10n_latam.check'
+    _description = 'Account payment check'
+    _check_company_auto = True
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+
+    payment_id = fields.Many2one(
+        'account.payment',
+        required=True,
+        ondelete='cascade',
+    )
+    operation_ids = fields.Many2many(
+        comodel_name='account.payment', relation='account_payment_account_payment_check_rel',
```

Comment:
relation='l10n_latam_check_account_payment_rel', 'check_id', 'payment_id',

Would be more logical and the extra fields (not sure about the param name) would be helpful for upgrade script

@maq-adhoc

Reply:
Done

Reply:
done now @jco-odoo